karma.conf.js

```diff
@@ -40,8 +40,8 @@ module.exports = (config) => {
             coverageIstanbulReporter: {
                 thresholds: {
                     global: {
-                        statements: 95,
-                        branches: 86,
+                        statements: 96,
+                        branches: 89,
```

Comment:
Mainly from if/else ignores, but it is nice to see this number slowly trend upwards...